src_erl/Communication_Layer/http_Nerlserver/src/Client/clientStatem.erl

```diff
@@ -114,16 +113,43 @@ waitforWorkers(cast, In = {stateChange,WorkerName,MissedBatchesCount}, State = #
 waitforWorkers(cast, In = {NewState}, State = #client_statem_state{myName = MyName, etsRef = EtsRef}) ->
   ets:update_counter(EtsRef, msgCounter, 1),
   ets:update_counter(EtsRef, infoIn, nerl_tools:calculate_size(In)),
-  % ?LOG_INFO("~p in waiting going to state ~p~n",[MyName, State]),
+  %% ?LOG_INFO("~p in waiting going to state ~p~n",[MyName, State]),
   Workers = ets:lookup_element(EtsRef, workersNames, ?ETS_KV_VAL_IDX),
   cast_message_to_workers(EtsRef, {NewState}),
   {next_state, waitforWorkers, State#client_statem_state{nextState = NewState, waitforWorkers = Workers}};
 
+waitforWorkers(cast, In ={worker_kill , Body}, State = #client_statem_state{waitforWorkers = WaitforWorkers,etsRef = EtsRef}) ->
+  %got kill command for a worker, kiil and take off waitforWorkers list
+  ets:update_counter(EtsRef, msgCounter, 1),
+  ets:update_counter(EtsRef, infoIn, nerl_tools:calculate_size(In)),
```

Comment:
infoIn? 
What is it? add a documentation please.
If it is a size of in message, then more informative name is required.